FlightPlanProject/UI/FlightPlanUI.cs

```diff
@@ -0,0 +1,314 @@
+
+    public void SetManeuverType(ManeuverType type)
+    {
+        maneuver_type = type;
+        maneuver_type_desc = BurnTimeOption.Instance.setOptionsList(type);
+    }
+
+    public string maneuver_type_desc;
+
+
+    FlightPlanPlugin plugin;
+
+    public ManualLogSource Logger = BepInEx.Logging.Logger.CreateLogSource("FlightPlanUI");
+
+    BodySelection body_selection;
+    BurnTimeOption burn_options;
+
+    int spacingAfterEntry = 5;
+
+    private void DrawEntryButton(string entryName, ref bool button, string buttonStr, string value, string unit = "")
+    {
+        GUILayout.BeginHorizontal();
+        UI_Tools.Label(entryName);
+        GUILayout.FlexibleSpace();
+        button = UI_Tools.SmallButton(buttonStr);
+        UI_Tools.Label(value);
+        GUILayout.Space(5);
+        UI_Tools.Label(unit);
+        GUILayout.EndHorizontal();
+        GUILayout.Space(spacingAfterEntry);
+    }
+
+    private void DrawEntry2Button(string entryName, ref bool button1, string button1Str, ref bool button2, string button2Str, string value, string unit = "")
+    {
+        GUILayout.BeginHorizontal();
+        UI_Tools.Console(entryName);
+        GUILayout.FlexibleSpace();
+        button1 = UI_Tools.SmallButton(button1Str);
+        button2 = UI_Tools.SmallButton(button2Str);
+        UI_Tools.Console(value);
+        GUILayout.Space(5);
+        UI_Tools.Console(unit);
+        GUILayout.EndHorizontal();
+        GUILayout.Space(spacingAfterEntry);
+    }
+
+    private double DrawLabelWithTextField(string entryName, double value, string unit = "")
+    {
+        GUILayout.BeginHorizontal();
+        UI_Tools.Label(entryName);
+        GUILayout.Space(10);
+
+        value = UI_Fields.DoubleField(entryName, value);
+
+        GUILayout.Space(3);
+        UI_Tools.Label(unit);
+        GUILayout.EndHorizontal();
+
+        GUILayout.Space(spacingAfterEntry);
+        return value;
+    }
+
+    public double DrawToggleButtonWithTextField(string runString, ManeuverType type, double value, string unit = "", string stopString = "")
+    {
+        GUILayout.BeginHorizontal();
+        if (stopString.Length < 1)
+            stopString = runString;
+
+
+        DrawToggleButton(runString, type);
+        GUILayout.Space(10);
+
+        value = UI_Fields.DoubleField(runString, value);
+
+        GUILayout.Space(3);
+        UI_Tools.Label(unit);
+        GUILayout.EndHorizontal();
+
+        GUILayout.Space(spacingAfterEntry);
+        return value;
+    }
+
+    public void DrawToggleButton(string txt, ManeuverType maneuveur_type)
+    {
+        bool active = maneuver_type == maneuveur_type;
+
+        bool result = UI_Tools.SmallToggleButton(active, txt, txt);
+        if (result != active)
+        {
+            if (!active)
+                SetManeuverType(maneuveur_type);
+            else
+                SetManeuverType(ManeuverType.None);
+        }
+    }
+
+    bool init_done = false;
+
+    void createTabs()
```

Comment:
add tab here but just adding new classes. we could perhaps separate the Own Orbit tabs intot 2 separates